main/lib/core/src/FileDeserializer.cc

```diff
@@ -10,7 +10,7 @@
 namespace eudaq {
   FileDeserializer::FileDeserializer(const std::string &fname, bool faileof,
                                      size_t buffersize)
-      : m_file(0), m_faileof(faileof), m_buf(buffersize), m_start(&m_buf[0]),
+    : m_filename(fname), m_file(0), m_faileof(faileof), m_buf(buffersize), m_start(&m_buf[0]),
```

Comment:
`fname`-> `m_filename`?

Reply:
not sure I understand this comment. I added `m_filename`. `m_file` still exists.

Reply:
attached the comment to the wrong line, sorry:

    m_file = fopen(fname.c_str(), "rb");

Reply:
done.